.travis.yml

```diff
@@ -0,0 +1,48 @@
+sudo: false
+dist: trusty
+language: node_js
+branches:
+  only:
+    - master
+jobs:
+  fast_finish: true
+  allow_failures:
+    - env: WEBPACK_VERSION=canary
+  include:
+    - &test-latest
+      stage: Webpack latest
+      node_js: 6
+      env: WEBPACK_VERSION=latest JOB_PART=test
+      script: npm run travis:$JOB_PART
+    - <<: *test-latest
+      node_js: 4.3
+      env: WEBPACK_VERSION=latest JOB_PART=test
+      script: npm run travis:$JOB_PART
+    - <<: *test-latest
+      node_js: 8
+      env: WEBPACK_VERSION=latest JOB_PART=lint
+      script: npm run travis:$JOB_PART
+    - <<: *test-latest
+      node_js: 8
+      env: WEBPACK_VERSION=latest JOB_PART=coverage
+      script: npm run travis:$JOB_PART
+      after_success: 'bash <(curl -s https://codecov.io/bash)'
+    - stage: Webpack canary
+      before_script: npm i --no-save git://github.com/webpack/webpack.git#master
+      script: npm run travis:$JOB_PART
+      node_js: 8
+      env: WEBPACK_VERSION=canary JOB_PART=test
+before_install:
+  - 'if [[ `npm -v` != 5* ]]; then npm i -g npm@^5.0.0; fi'
```

Comment:
`'if [[ `npm -v` != 5* ]]; then npm i -g npm@latest; fi'`

Should fix the problem